experimental/Schemes/Resolution_structure.jl

```diff
@@ -1,54 +1,138 @@
-## Warnung: show auf desingMor geht noch nicht!!!
 export _desing_curve
 export  find_refinement_with_local_system_of_params
 
+##############################################################################
+## Concrete Type for normalization
+## very similar to CoveredSchemeMorphism, but allowing disjoint handling
+## of disjoint components
+##############################################################################
+@attributes mutable struct NormalizationMorphism{
+    DomainType<:AbsCoveredScheme,
+    CodomainType<:AbsCoveredScheme,
+    BaseMorphsimType
+   } <:AbsCoveredSchemeMorphism{
+                                 DomainType,
+                                 CodomainType,
+                                 Nothing,
+                                 NormalizationMorphism,
+                                }
+
+  X::DomainType
+  Y::CodomainType
+  f::CoveringMorphism
```

Comment:
Here you are essentially duplicating the implementation of `CoveredSchemeMorphism`.
This is maybe possible, but not necessary. After all you want a `CoveredSchemeMorphism` which knows a bit more. You can do as follows:
```suggestion
  f::CoveredSchemeMorphism{DomainType, CodomainType, BaseMorphismType}
```
then implement 
`underlying_scheme(f::NormalizationMorphism) = f.X`
and you are done.

Reply:
done